Source/Core/Core/PowerPC/JitArm64/JitArm64_LoadStorePaired.cpp

```diff
@@ -173,115 +173,116 @@ void JitArm64::psq_stXX(UGeckoInstruction inst)
 
   const bool have_single = fpr.IsSingle(inst.RS);
 
-  ARM64Reg VS = fpr.R(inst.RS, have_single ? RegType::Single : RegType::Register);
-
-  if (js.assumeNoPairedQuantize)
   {
```

Comment:
You don't need to add an additional scope for the sake of unlocking VS earlier. Unlocking it last will just move it past some other unlock operations, not past anything that emits instructions.